lib/IDE/CommentConversion.cpp

```diff
@@ -538,17 +544,25 @@ class DoxygenConverter : public MarkupASTVisitor<DoxygenConverter> {
 
 public:
   DoxygenConverter(llvm::raw_ostream &OS, unsigned Indent = 0)
-    : OS(OS), Indent(Indent), IsFreshLine(true) {
+    : OS(OS), Indent(Indent), IsFreshLine(true), IsEmptyComment(true) {
     printIndent();
-    print("/**");
-    printNewline();
-    indent();
+    printOpeningComment();
+    // Add one space after starting comment
+    indent(1);
```

Comment:
Instead of `indent(1)`/`dedent()` in constructor/destructor, can't we always `+1` in `printIndent`?
```cpp
  void printIndent() {
    for (unsigned i = 0; i < Indent + 1; ++i) {
```